src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -687,6 +687,23 @@ default TypeElement getOutermostTypeElement(Element e) {
      *          elements.getTypeElement("C")); }
      * </blockquote>
      *
+     * Consistent with the usage of the {@link Override @Override} annotation, if an
+     * interface declares a method override-equivalent to a {@code
+     * public} method of {@link Object java.lang.Object}, this method treats
+     * any such interface methods as overriding the corresponding {@code
+     * Object} method; example:
```

Comment:
1. It doesn't help comprehension when "method" means two things in a sentence at the same time: (i) ExecutableElement that models a method and (ii) the method the doc comment belongs to (i.e. Elements.overrides) and that takes ExecutableElement as its argument(s). 
I understand that it's always tricky in `javax.lang.model` domain, but still wonder if the sentence can be tweaked a bit.

2. "example:"? Would "for example:" be better?